python/tk_multi_publish2/publish_tree_widget/custom_widget_context.py

```diff
@@ -56,5 +57,22 @@ def set_status(self, status):
         """
         pass
 
-    def set_checkbox_value(self, state):
```

Comment:
The definition of this function prevents checkbox from being initialized/set properly. Removing the void definition here to use the version implemented in parent class.